social_distribution/posts/models.py

```diff
@@ -56,4 +56,5 @@ class VisibilityType(models.TextChoices):
         choices=VisibilityType.choices,
         default=VisibilityType.PUBLIC
     )
+    visibleTo = models.CharField(max_length=200, blank=True)
```

Comment:
I would suggest you make this a ManyToMany rela so as we can get a drop-down list of poeple you can share with

Reply:
we shouldn't have user input any kind of ID because that is not intuitive plus once we connect to other nodes, we can't know other nodes' IDs whatsoever.

Reply:
> we shouldn't have user input any kind of ID because that is not intuitive plus once we connect to other nodes, we can't know other nodes' IDs whatsoever.

Isn't only author id is unique?